flink-table/flink-table-common/src/test/java/org/apache/flink/table/types/inference/InputTypeStrategiesTest.java

```diff
@@ -58,15 +42,16 @@
 import static org.apache.flink.table.types.inference.InputTypeStrategies.or;
 import static org.apache.flink.table.types.inference.InputTypeStrategies.sequence;
 import static org.apache.flink.table.types.inference.InputTypeStrategies.varyingSequence;
-import static org.apache.flink.util.CoreMatchers.containsCause;
-import static org.hamcrest.CoreMatchers.equalTo;
-import static org.junit.Assert.assertThat;
 
 /**
  * Tests for built-in {@link InputTypeStrategies}.
  */
 @RunWith(Parameterized.class)
```

Comment:
We can move this annotation to the test base.

Reply:
I thought the annotation cannot be on the abstract class. Or to say it differently that the `Parameter` can be in a separate class than `Parameters`.

Changed that.